src/findex/sqlite.ts

```diff
@@ -10,33 +10,38 @@ import {
 
 /**
  * @param db the SQLite3 connection
+ * @param init create indexes-tables
  * @returns the callbacks
  */
-export function callbacksExamplesBetterSqlite3(db: Database): {
+export function callbacksExamplesBetterSqlite3(
+  db: Database,
+  init?: boolean,
+): {
   fetchEntries: FetchEntries
   fetchChains: FetchChains
   upsertEntries: UpsertEntries
   insertChains: InsertChains
 } {
-  db.prepare(
-    "CREATE TABLE entries (uid BLOB PRIMARY KEY, value BLOB NOT NULL)",
-  ).run()
-  db.prepare(
-    "CREATE TABLE chains (uid BLOB PRIMARY KEY, value BLOB NOT NULL)",
-  ).run()
-
+  if (typeof init === "undefined" || init) {
+    db.prepare(
+      "CREATE TABLE entry_table (uid BLOB PRIMARY KEY, value BLOB NOT NULL)",
+    ).run()
+    db.prepare(
+      "CREATE TABLE chain_table (uid BLOB PRIMARY KEY, value BLOB NOT NULL)",
+    ).run()
+  }
   //
   // Prepare some useful SQL requests on different databases
   // `prepare` a statement is a costly operation we don't want to do on every line (or in every callback)
   //
   const upsertIntoChainsTableStmt = db.prepare(
-    `INSERT OR REPLACE INTO chains (uid, value) VALUES(?, ?)`,
+    `INSERT OR REPLACE INTO chain_table (uid, value) VALUES(?, ?)`,
```

Comment:
Why do you need to rename the tables? I don't like obvious suffix like `table` for a SQL table neither singular form for tables hosting multiple entries. But if you have a good reason to change this :-D

Reply:
These table names are used in multiple place in the examples (to compute stats for exemple). I think it's better to accept override for table name for your regressions vectors.

Reply:
well, I need the same SQL scheme in every language. 
However, I can change those names everywhere.

Reply:
You could add `entries_table_name` and `chains_table_name` to the function?

Reply:
thanks